### PR TITLE
Update edit_mongo_one_association_script.html.twig

### PR DESCRIPTION
**Pull request by goncziakos**
I am targeting this branch, because {reason}.

Closes #{put_issue_number_here}

## Changelog

```markdown
### Added
- Added some `Class::newMethod` to do great stuff

### Changed

### Deprecated

### Removed

### Fixed

### Security
```

## To do

- [ ] Update the tests
- [ ] Update the documentation
- [ ] Add an upgrade note

### CODE REVIEW

**Comment by greg0ire** [comment]
Hey there!

It looks like you did not fill the pull request template properly. Maybe you
were confused, so let me walk you through this step by step.

# Explaining why you target this branch

> I am targeting this branch, because {reason}.

Here, you are supposed to replace `{reason}` with something like:

- "there is a BC-break" if you picked the master branch
- "there is no BC-break" if you picked the stable branch

# Searching and finding related issues

> Closes #{put_issue_number_here}

Please have a look at the list of issues and pull requests, and look for issues
that will be fixed, and pull requests that will become irrelevant should we
merge your PR. Each issue / pull request has a number shown right next to the
title. Kindly replace `{put_issue_number_here}` with whatever issue or PR is
relevant. Of course, you may specify several numbers.
On merge, [these issues and PRs will automatically get
closed](https://help.github.com/articles/closing-issues-using-keywords/).
If there is no issue or pull request that fits there, just remove the sentence
entirely.

# Filling the changelog

That one is really important to get right, because it will be used to craft a
release changelog such as [this
one](https://github.com/sonata-project/SonataAdminBundle/releases/tag/3.23.0).
The first thing you need to know about this is that if your pull request is
pedantic, meaning it does not fix a bug for the user, or adds a new feature, it
is not needed. This is typically the case for PRs that fix the build or add
some documentation.

3 important things:

1. Make sure to fill the changelog section with something directed at the end
user, that answers the question "What benefit do I get from upgrading?".
2. Make sure to remove empty sections.
3. Make sure to keep the markdown markup.

## Filling the TODO list

If you think your PR is ready, just remove it. If you have still some things
left to do, customize it.

## Filling the Subject

Last but not least, describe your PR as well as you can. Bear in mind that some
reviewers do not even use the library they are maintaining, so try to be
crystal clear.

File: src/Resources/views/CRUD/edit_mongo_one_association_script.html.twig [hunk 1]
```diff
@@ -52,11 +52,14 @@ This code manage the one-to-many association field popup
...
                 // Maintain state of file inputs
                 $oldForm.find('input[type="file"]').each(function(){
-                    var id = '#'$(this).attr('id');
+                    var id = '#'+$(this).attr('id');
                     $newForm.find(id).replaceWith($(this));
                 });
 
                 $oldForm.replaceWith($newForm);  // replace the html
+
+                Admin.shared_setup(jQuery('#field_container_{{ id }}'));
```
Comment (OskarStark) [inline src/Resources/views/CRUD/edit_mongo_one_association_script.html.twig]: can you explain, why are you doing this?

Comment (goncziakos) [reply src/Resources/views/CRUD/edit_mongo_one_association_script.html.twig]: Hello! In inline edit mode, ajax request return new form row and this line builds and init costum input type javascript tools. 
Example checkbox style: https://i.stack.imgur.com/UwI5L.png
My English is not good, I'm sorry.

**Comment by jordisala1991** [comment]
Aren't we moving to AdminBundle for the templates? cc @kunicmarko20

If this is the case, we might stop it here, wait for the move, and then migrate this bundle too

**Comment by kunicmarko20** [comment]
Yes, `PHPCR` is already using templates from `AdminBundle` and we are now migrating `DoctrineORM` to start using it, so this bundle can also migrate.

**Comment by jordisala1991** [comment]
will you be able to help this bundle moving templates to admin eventually?

**Comment by kunicmarko20** [comment]
I can provide a PR like for `DoctrineORM` if you want, but I am not using this bundle or mongo so I can't really test if everything is ok.

File: src/Resources/views/CRUD/edit_mongo_one_association_script.html.twig (hunk 1 above)
Comment (enleur) [reply src/Resources/views/CRUD/edit_mongo_one_association_script.html.twig]: Seems lie copy-paste from https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/views/CRUD/Association/edit_one_script.html.twig#L61

**Comment by jordisala1991** [comment]
Duplicate of #270

**Pull request closed without merging**